lib/coradoc/document/table.rb

```diff
@@ -1,18 +1,62 @@
 module Coradoc
   module Document
     class Table
-      attr_reader :title, :rows
+      attr_reader :title, :rows, :content, :id
 
-      def initialize(title, rows)
+      def initialize(title, rows, options = {})
         @rows = rows
         @title = title
+        @id = options.fetch(:id, nil)
+        @attrs = options.fetch(:attrs, '')
+      end
+
+
+      def to_adoc
+        anchor = @id ? "[[#{@id}]]\n" : ""
+        attrs = @attrs
+        title = @title
+        content = Coradoc::Generator.gen_adoc(@rows)
```

Comment:
Shouldn't `@rows` already be Row objects? If so, why can't we call `@rows.to_adoc`?